projects/espressif/esp32/make/aws_tests/.project

```diff
@@ -2876,9 +2876,14 @@
 			<locationURI>virtual:/virtual</locationURI>
 		</link>
 		<link>
-			<name>libraries/c_sdk/aws/defender/test/aws_iot_tests_defender_api.c</name>
+			<name>libraries/c_sdk/aws/defender/test/unit/aws_iot_tests_defender_unit.c</name>
 			<type>1</type>
-			<locationURI>PARENT-1-BASE_DIR_ROOT/libraries/c_sdk/aws/defender/test/aws_iot_tests_defender_api.c</locationURI>
+			<locationURI>AWS_IOT_MCU_ROOT/libraries/c_sdk/aws/defender/test/unit/aws_iot_tests_defender_unit.c</locationURI>
```

Comment:
Why did this change from `PARENT-1-BASE_DIR_ROOT` to `AWS_IOT_MCU_ROOT`?

Reply:
Good catch. Amended.

Reply:
See 38bebfb84bc9